### PR TITLE
update getting started

### PR DESCRIPTION
**Pull request by R-Lawton**

### CODE REVIEW

File: docs/getting-started.md [hunk 1]
```diff
@@ -1,31 +1,44 @@
...
 | `MGC_AWS_REGION`             | `eu-west-1`                 | AWS Region                                                     |
-| `MGC_SUB_DOMAIN`             | `myapp.jbloggs.hcpapps.net` | AWS Region                                                     |
+| `MGC_SUB_DOMAIN`             | `myapp.jbloggs.hcpapps.net` | Desired Sub domain                                             |
+
+
+### GCP
+
+   | Env Var                 | Example Value          | Description                                                    |
+   |-------------------------|------------------------|----------------------------------------------------------------|
+   | `GOOGLE`     | `{"client_id": "00000000-00000000000000.apps.googleusercontent.com","client_secret": "d-FL95Q00000000000000","refresh_token": "00000aaaaa00000000-AAAAAAAAAAAAKFGJFJDFKDK","type": "authorized_user"}` |  This is the JSON created from either the credential created by the cli or the json from the Service account             |
```
Comment (jasonmadigan) [inline docs/getting-started.md:33]:
```suggestion
   | `GOOGLE`     | `{"client_id": "00000000-00000000000000.apps.googleusercontent.com","client_secret": "d-FL95Q00000000000000","refresh_token": "00000aaaaa00000000-AAAAAAAAAAAAKFGJFJDFKDK","type": "authorized_user"}` |  This is the JSON created from either the credential created by the Google Cloud CLI or the JSON from the Service account             |
```

Comment (jasonmadigan) [reply docs/getting-started.md:33]: (small nit)

File: docs/getting-started.md [hunk 2]
```diff
@@ -1,31 +1,44 @@
...
+   | `PROJECT_ID` | `my_project_id`   | ID to the google project |
+   | `ZONE_NAME`       | `jbloggs-google`   | Zone name                          |
+   | `ZONE_DNS_NAME` | `jbloggs.google.hcpapps.net`   | DNS name                        |
+   | `LOG_LEVEL`              | `1`                     | Log level for the Controller                          |
 
 >Alternatively, to set defaults, add the above environment variables to your `.zshrc` or `.bash_profile`.
 
-### Set Up Clusters and Kuadrant
+### Set Up Clusters and Multicluster Gateway Controller
```
Comment (jasonmadigan) [inline docs/getting-started.md:41]: I'd suggest reverting back - the quickstart now installs not only the MGC, but also all of the service protection components as well. So I think setup cluster + Kuadrant might make more sense?

File: docs/getting-started.md [hunk 3]
```diff
@@ -35,5 +48,5 @@ curl https://raw.githubusercontent.com/kuadrant/multicluster-gateway-controller/
 
 ### What's Next
 
-* [Multicluster Gateways walkthrough](how-to/multicluster-gateways-walkthrough.md)
-* [Kuadrant Documentation](https://docs.kuadrant.io).
+Now that you have two Kind clusters configured with the Multicluster Gateway Controller installed you are ready to begin [the Multicluster Gateways walkthrough.](how-to/multicluster-gateways-walkthrough.md)
```
Comment (jasonmadigan) [inline docs/getting-started.md:51]: `the Multicluster Gateway Controller` -> `Kuadrant`

**Review by jasonmadigan — CHANGES_REQUESTED** [review]

**Comment by jasonmadigan** [comment]
few small comments

File: docs/getting-started.md (hunk 3 above)
Comment (R-Lawton) [reply docs/getting-started.md:51]: ill update the links to

**Comment by jasonmadigan** [comment]
Not sure why tide isn't adding `approved` label for this repo...

**Pull request merged**